### PR TITLE
[MINOR][SQL] Fix the error message typo in QueryExecutionErrors.scala

### PR DESCRIPTION
**Pull request by sadikovi**
### What changes were proposed in this pull request?

Very small change to fix the typo in the error message when we could not merge two DecimalType values.

### Why are the changes needed?
Fixes typo.

### Does this PR introduce _any_ user-facing change?

No

### How was this patch tested?

Existing unit and integration tests.

### CODE REVIEW

**Comment by sadikovi** [comment]
@HyukjinKwon @beliefer could you review? It is a very small error message change. Thank you.

**Review by wangyum — APPROVED** [review]

**Review by HyukjinKwon — APPROVED** [review]

File: sql/catalyst/src/main/scala/org/apache/spark/sql/errors/QueryExecutionErrors.scala [hunk 1]
```diff
@@ -1894,4 +1894,3 @@ object QueryExecutionErrors {
     new RuntimeException("Unable to convert timestamp of Orc to data type 'timestamp_ntz'")
   }
 }
-
```
Comment (beliefer) [inline sql/catalyst/src/main/scala/org/apache/spark/sql/errors/QueryExecutionErrors.scala:1897]: @wangyum preserve the end line?

**Review by beliefer — APPROVED** [review]

File: sql/catalyst/src/main/scala/org/apache/spark/sql/errors/QueryExecutionErrors.scala [hunk 2]
```diff
@@ -1894,4 +1894,3 @@ object QueryExecutionErrors {
     new RuntimeException("Unable to convert timestamp of Orc to data type 'timestamp_ntz'")
   }
 }
```
Comment (HyukjinKwon) [inline sql/catalyst/src/main/scala/org/apache/spark/sql/errors/QueryExecutionErrors.scala:1896]:
```suggestion
}

```

**Comment by HyukjinKwon** [comment]
I addressed it by myself.

Merged to master.

File: sql/catalyst/src/main/scala/org/apache/spark/sql/errors/QueryExecutionErrors.scala (hunk 1 above)
Comment (sadikovi) [reply sql/catalyst/src/main/scala/org/apache/spark/sql/errors/QueryExecutionErrors.scala]: No, we should not keep an empty line! We do keep \n at the end of the file, but not an empty line!

**Comment by sadikovi** [comment]
Thanks for merging, but I disagree with an empty line change, it should have been removed - this is different from \n at the end of the file!

**Comment by HyukjinKwon** [comment]
Actually it does already a newline:
Screen Shot 2021-11-25 at 10 37 22 AM

That newline was another newline :-).

**Comment by sadikovi** [comment]
Yes, that's right. The new line at the end should have been removed which is what my commit did. There should be no empty lines at the end of the file but \n.

**Comment by HyukjinKwon** [comment]
oh yeah. true!

**Comment by sadikovi** [comment]
Well, it does not matter now.

**Comment by HyukjinKwon** [comment]
it should better have been removed I think, yeah. sorry for pushing it without asking 🙏

**Comment by sadikovi** [comment]
It is all good, no worries 👍.

**Pull request closed without merging**